jsamchineme/send-it
# Implement page transition

#### What does this PR do?
- Implement page transition after re-render
#### Description of Task to be completed?
- Make the view re-render and apply slight fade in effect on page transition
#### How should this be manually tested?
- pull branch and run _npm run start:dev_
- checkout the app on `localhost:3000` and see the above effect at play
#### Any background context you want to provide?
- None
#### What are the relevant pivotal tracker stories?
- https://www.pivotaltracker.com/story/show/162200441
#### Screenshots (if appropriate)
None
#### Questions:`
None
